caps/templates/about_data.html

```diff
@@ -54,6 +54,10 @@ <h2 class="mt-5 mb-4" id="csv">CSV files</h2>
 
         <p>All CSV files include the council GSS code, and a three letter council code you can use with the API, to enable matching data across files, and with other sources.</p>
 
+         <h2 class="mb-4" id="licensing">Licensing</h2>
+
+        <p>The API data and CSV files are licensed under a <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>. The zip file is unlicensed as it contains work issued under many licenses.</p>
```

Comment:
When talking about the zip file, maybe worth saying "work issued by local authorities under multiple licenses" ? To make it clear whose work we’re talking about?

Also, if we’re capitalising "CSV" we should probably capitalise "ZIP".